diff --git a/config.def.h b/config.def.h
index 97b7739..e6cf3fa 100644
--- a/config.def.h
+++ b/config.def.h
@@ -21,6 +21,11 @@ static const char *colors[][3]      = {
 	/*               fg         bg         border   */
 	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
 	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeStatus]  = { col_gray3, col_gray1,  "#000000"  }, // Statusbar right {text,background,not used but cannot be empty}
+	[SchemeTagsSel]  = { col_gray4, col_cyan,  "#000000"  }, // Tagbar left selected {text,background,not used but cannot be empty}
+    [SchemeTagsNorm]  = { col_gray3, col_gray1,  "#000000"  }, // Tagbar left unselected {text,background,not used but cannot be empty}
+    [SchemeInfoSel]  = { col_gray4, col_cyan,  "#000000"  }, // infobar middle  selected {text,background,not used but cannot be empty}
+    [SchemeInfoNorm]  = { col_gray3, col_gray1,  "#000000"  }, // infobar middle  unselected {text,background,not used but cannot be empty}
 };
 
 /* tagging */
diff --git a/config.h b/config.h
index f4fbe8e..e260a73 100644
--- a/config.h
+++ b/config.h
@@ -43,11 +43,11 @@ static const char *colors[][3]      = {
 	/*               fg         bg         border   */
 	[SchemeNorm]     = { tn_foreground, tn_background, tn_background },
 	[SchemeSel]      = { tn_foreground, tn_background, tn_green  },
-        //[SchemeStatus]   = { tn_foreground, tn_background, "#000000"  }, // Statusbar right {text, background, not used}
-	//[SchemeTagsSel]  = { tn_green,      tn_background, "#000000"  }, // Tagbar left selected {text, background, not used}}
-        //[SchemeTagsNorm] = { tn_foreground, tn_background, "#000000"  }, // Tagbar left unselected {text, background}
-        //[SchemeInfoSel]  = { tn_foreground, tn_background, "#000000"  }, // infobar middle  selected {text, background, not used}
-        //[SchemeInfoNorm] = { tn_foreground, tn_background, "#000000"  }, // infobar middle  unselected {text, background, not used}
+        [SchemeStatus]   = { tn_foreground, tn_background, "#000000"  }, // Statusbar right {text, background, not used}
+	[SchemeTagsSel]  = { tn_green,      tn_background, "#000000"  }, // Tagbar left selected {text, background, not used}}
+        [SchemeTagsNorm] = { tn_foreground, tn_background, "#000000"  }, // Tagbar left unselected {text, background}
+        [SchemeInfoSel]  = { tn_foreground, tn_background, "#000000"  }, // infobar middle  selected {text, background, not used}
+        [SchemeInfoNorm] = { tn_foreground, tn_background, "#000000"  }, // infobar middle  unselected {text, background, not used}
 };
 
 /* tagging */
diff --git a/dwm.c b/dwm.c
index ba498f9..c17420f 100644
--- a/dwm.c
+++ b/dwm.c
@@ -59,7 +59,7 @@
 
 /* enums */
 enum { CurNormal, CurResize, CurMove, CurLast }; /* cursor */
-enum { SchemeNorm, SchemeSel }; /* color schemes */
+enum { SchemeNorm, SchemeSel, SchemeStatus, SchemeTagsSel, SchemeTagsNorm, SchemeInfoSel, SchemeInfoNorm }; /* color schemes */
 enum { NetSupported, NetWMName, NetWMState, NetWMCheck,
        NetWMFullscreen, NetActiveWindow, NetWMWindowType,
        NetWMWindowTypeDialog, NetClientList, NetLast }; /* EWMH atoms */
@@ -759,9 +759,9 @@ drawstatusbar(Monitor *m, int bh, char* stext) {
 	w += 2; /* 1px padding on both sides */
 	ret = x = m->ww - w;
 
-	drw_setscheme(drw, scheme[LENGTH(colors)]);
-	drw->scheme[ColFg] = scheme[SchemeNorm][ColFg];
-	drw->scheme[ColBg] = scheme[SchemeNorm][ColBg];
+	drw_setscheme(drw, scheme[SchemeStatus]);
+	drw->scheme[ColFg] = scheme[SchemeStatus][ColFg];
+	drw->scheme[ColBg] = scheme[SchemeStatus][ColBg];
 	drw_rect(drw, x, 0, w, bh, 1, 1);
 	x++;
 
@@ -859,7 +859,7 @@ drawbar(Monitor *m)
 	x = 0;
 	for (i = 0; i < LENGTH(tags); i++) {
 		w = TEXTW(tags[i]);
-		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeSel : SchemeNorm]);
+		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeTagsSel : SchemeTagsNorm]);
 		drw_text(drw, x, 0, w, bh, lrpad / 2, tags[i], urg & 1 << i);
 		if (occ & 1 << i)
 			drw_rect(drw, x + boxs, boxs, boxw, boxw,
@@ -868,17 +868,17 @@ drawbar(Monitor *m)
 		x += w;
 	}
 	w = blw = TEXTW(m->ltsymbol);
-	drw_setscheme(drw, scheme[SchemeNorm]);
+	drw_setscheme(drw, scheme[SchemeTagsNorm]);
 	x = drw_text(drw, x, 0, w, bh, lrpad / 2, m->ltsymbol, 0);
 
 	if ((w = m->ww - tw - x) > bh) {
 		if (m->sel) {
-			drw_setscheme(drw, scheme[m == selmon ? SchemeSel : SchemeNorm]);
+			drw_setscheme(drw, scheme[m == selmon ? SchemeInfoSel : SchemeInfoNorm]);
 			drw_text(drw, x, 0, w, bh, lrpad / 2, m->sel->name, 0);
 			if (m->sel->isfloating)
 				drw_rect(drw, x + boxs, boxs, boxw, boxw, m->sel->isfixed, 0);
 		} else {
-			drw_setscheme(drw, scheme[SchemeNorm]);
+			drw_setscheme(drw, scheme[SchemeInfoNorm]);
 			drw_rect(drw, x, 0, w, bh, 1, 1);
 		}
 	}
